INSTALLATION.md

```diff
@@ -13,6 +13,52 @@ cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-conf
 cilium hubble enable
 ```
 
+Alternatively, you can use Helm to install Cilium Service Mesh. For example, to install on GKE:
+
+```
+# Specify your cluster name and zone.
+CLUSTER_NAME=my-cluster
+CLUSTER_ZONE=us-west2-a
+
+NATIVE_CIDR="$(gcloud container clusters describe $CLUSTER_NAME --zone $CLUSTER_ZONE --format 'value(clusterIpv4Cidr)')"
+git clone git@github.com:cilium/cilium.git
```

Comment:
Do they really need to clone the whole repo to use Helm? If we're not already, we could be publishing just the Helm chart

Reply:
we need the chart from `origin/beta/service-mesh` branch, which contains some changes needed for service mesh to work. let me check and see if we can publish a chart from this branch.